modules/apps/commerce/commerce-warehouse-web/src/main/java/com/liferay/commerce/warehouse/web/internal/portlet/action/EditCommerceInventoryWarehouseItemMVCActionCommand.java

```diff
@@ -111,7 +110,4 @@ protected void doProcessAction(
 	private CommerceInventoryWarehouseItemService
 		_commerceInventoryWarehouseItemService;
 
-	@Reference
-	private Portal _portal;
```

Comment:
@JanisZhang Please record this one, `SessionErrors` has a lot of methods calling `PortalUtil`.

Reply:
Got it, thanks